lib/blocks.php

```diff
@@ -56,6 +56,7 @@ function gutenberg_reregister_core_block_types() {
 					'cover.php'                     => 'core/cover',
 					'latest-comments.php'           => 'core/latest-comments',
 					'latest-posts.php'              => 'core/latest-posts',
+					'loginout.php'                  => 'core/loginout',
```

Comment:
I see the name is coming from https://developer.wordpress.org/reference/functions/wp_loginout/ but we should rather fix it and make it easier to read for the block. Did you consider alternative names?

Reply:
Any suggestions for alt names?

Reply:
`user-authentication` is an alternative that comes to mind, but it's inaccurate and I'm not sure it would be easier that `loginout` 🤔